official-antistasi-community/A3-Antistasi
# Meer 3CB tweaks

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    Many quality tweaks to the 3CB templates. See commit messages for details.
Summary: Corrected faction names, incorrect radio classnames, made temperate BAF actually temperate, multiple balance and unit selection tweaks.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [ ] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Load the various maps with 3CB, check the starting LR and police units. Check blu low tier garrison troops on arid.
********************************************************
Notes:
